### ISSUE TITLE
Question about the missing cocomask files and the released model weights

### ISSUE THREAD

**Issue by weiweilike123** [YinKangSheng/UG-ICM#7]
Great work! Thank you very much for sharing the code and pretrained weights of UG-ICM.

However, I could not find the cocomask files required for the second-stage training (train_mask.py) in the released project or weight package. Could you please provide the cocomask data or share the generation procedure for these mask files?

Besides, I would like to clarify the four released weight files (1.pth.tar–4.pth.tar). Are these weights the final Stage 2 models obtained by fine-tuning from Stage 1 checkpoints with different λ values (different compression levels)? Or do they correspond to other training stages/configurations?

Thank you very much for your help!

**Comment by hellobo2802** [comment]
你好，你是如何复现该论文的推理流程/训练流程的？
作者在论文中指出训练使用的是coco2017-train的子集80000张图像（未给出list.txt划分），推理也是使用的是各个数据集（除Kodak）的测试集的子集（未给出子集划分依据和标准）。如何复现论文中的推理/训练指标结果？
望请告知，感谢回复。
